doc/api/hdr.md

```diff
@@ -0,0 +1,90 @@
+# HDR support ##################################################################
```

Comment:
Is this related to this PR (trick mode) ? or is concerning the HDR ?

Reply:
It's because the branch is either not properly rebased on the last `next` branch or other git/github - specific issues.

You can see in the linked commits that some commits from `next` are included here. I will try to remove those from there.